### ISSUE TITLE
Setting up Data Service HOWTO

### ISSUE THREAD

**Issue by asmodehn** [garethdmm/gryphon#74]
I have been trying to setup the data service on my machine, but it looks like there are some information missing from [the docs](https://gryphon.readthedocs.io/en/latest/data_service.html).

Questions that comes to mind : 
- how do I setup the GDS database ? There is a setting  should put in the configuration file, but no info about that.
- where do I find `foreman` ? It doesn't seem to be installed by the requirements... something missing somewhere, or not uptodate ?
- why do I need to `cd gryphon/data_service/` it is a python package, isn't it ? no entry point ?
- whats `emerald` ?
- Should I attempt use `runt.py` directly or not ?
- the [README](https://github.com/garethdmm/gryphon/blob/master/gryphon/data_service/README.md)  is empty...

So I m a bit confused as you can see...

**Comment by garethdmm** [comment]
Yeah this is a blind-spot in the docs right now, just because it's a more advanced feature so it wasn't a priority to document for launch. I can get you started on these questions.

The GDS database is set up the same way as dashboards/trading. Make a new database, point your `GDS_DB_CRED` environment variable at it, and run `gryphon-exec run_migrations --database gds`

Foreman is a ruby gem, so it is usually installed with a command like `gem install foreman`.

There currently isn't a built-in entrypoint for GDS because it is a more complicated service than the strategy engine or other things accessible through `gryphon-exec`. GDS uses foreman to orchestrate multiple processes at once, and foreman requires a procfile in the current working directory. It was expected that users would want to edit their own procfile, so we left it without a simple entrypoint in the package.

`emerald` is an old name for GDS which still shows up in the code some places. They are the same thing.

It's actually fine to run runt.py on it's own for short to medium periods of time at once. You'll get orderbook data into redis that strategies can use, but the consumers won't be running so that data won't get into the GDS database.
